src/pages/GetAssistancePage.tsx

```diff
@@ -1,61 +1,50 @@
-import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
+import type {RouteProp} from '@react-navigation/native';
 import React from 'react';
 import {ScrollView, View} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx/lib/types';
 import HeaderWithBackButton from '@components/HeaderWithBackButton';
 import * as Expensicons from '@components/Icon/Expensicons';
 import * as Illustrations from '@components/Icon/Illustrations';
+import type {MenuItemWithLink} from '@components/MenuItemList';
 import ScreenWrapper from '@components/ScreenWrapper';
 import Section from '@components/Section';
 import Text from '@components/Text';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
-import compose from '@libs/compose';
 import Navigation from '@libs/Navigation/Navigation';
 import * as Link from '@userActions/Link';
 import * as Report from '@userActions/Report';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
+import type {Route} from '@src/ROUTES';
+import type {Account} from '@src/types/onyx';
 
-const propTypes = {
-    /** Route object from navigation */
-    route: PropTypes.shape({
-        params: PropTypes.shape({
-            /** The task ID to request the call for */
-            taskID: PropTypes.string,
-        }),
-    }).isRequired,
-
+type GetAssistanceOnyxProps = {
     /** The details about the account that the user is signing in with */
-    account: PropTypes.shape({
-        /** URL to the assigned guide's appointment booking calendar */
-        guideCalendarLink: PropTypes.string,
-    }),
-
-    ...withLocalizePropTypes,
+    account: OnyxEntry<Account>;
 };
 
-const defaultProps = {
-    account: {
-        guideCalendarLink: null,
-    },
+type GetAssistancePageProps = GetAssistanceOnyxProps & {
+    /** Route object from navigation */
+    route: RouteProp<{params: {backTo: Route}}>;
 };
```

Comment:
@pecanoro Unfortunately I missed this PR before the merge, this is not a big deal but this part should look a bit different:
```ts
type GetAssistancePageProps = GetAssistanceOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.GET_ASSISTANCE>;
```
And improved type:
```ts
// src/navigation/types.ts
[SCREENS.GET_ASSISTANCE]: {
    backTo: Route;
};
```

Reply:
@blazejkustra 

Noted, I'll catch similar cases in the future. Do you have some other concerns about this PR?

@DylanDylann 

Would you file a follow-up PR with this typing improvement?

Reply:
I will create a quick PR to update

Reply:
@cubuspl42 The PR is created [here](https://github.com/Expensify/App/pull/34715), tested on web